### ISSUE TITLE
Question regarding the training time discrepancy (FreePose-S, 81 frames)

### ISSUE THREAD

**Issue by xw-design1** [Lxg-233/FreePose#1]
Hi authors,

Thanks for your excellent work and for open-sourcing the code! I am currently trying to reproduce the results of FreePose, but I have encountered a significant discrepancy in training time and would like to seek your advice.

In your paper/documentation, it is mentioned that the training time is around 9 hours on a single RTX 4090. However, when I train the FreePose-S (81 frames) model using your provided configurations, it takes me about 22 hours on a single RTX 3090, and roughly 17 hours when I switch to a single RTX 4090.

Here is my hardware and software setup:

GPU: Single RTX 3090 / RTX 4090 (Cloud server instance)

Environment: PyTorch 2.1.2, CUDA 11.8

Dataset: Human3.6M (81 frames setting)

Command: CUDA_VISIBLE_DEVICES=0 python train.py --config configs/h36m/MotionAGFormer-small.yaml --seed 42

Could you please share some details about your experimental configuration? Specifically:

Did you set torch.backends.cudnn.benchmark = True and turn off the deterministic mode during your 9-hour training?

What num_workers setting did you use for the DataLoader?

Are there any specific I/O optimizations (like reading data directly from memory/RAM disk) you applied to speed up the DWT and sequence loading?

Any insights or suggestions to accelerate the training process would be greatly appreciated. Thank you very much for your time!

Best regards.

**Comment by Lxg-233** [comment]
* 你好，感谢你对我工作的认可。
* * 关于模型训练，我并没有进行特殊处理，超参数均在configs的配置里。
* * 你在使用RTX4090复现时出现了训练耗时很长的问题，是不是其他训练任务也在云服务器上导致的。

**Comment by xw-design1** [comment]
感谢作者的回复。
我在云服务器里面再试一试。
